.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: ruby
+rvm:
+  - "2.3.0"
+
+
+script:
+- bundle exec rake db:create
+- bundle exec rake db:migrate
+
+
+deploy:
+  provider: heroku
+  api_key:
+    secure: "c2471f4b-ebb1-41a5-b8d5-487f86458829"
```

Comment:
api_key is being exposed. It is not safe. We need find another way to change it. In the past, we add a new template file (might be named .template.secret.yml) with env variables in somewhere of server. In the processing deployment, template file could be copied to application config folder and named secret.yml. So this file could only be saved in server. But I'm sorry I don't know how to do it in Heroku. I need search.